assets/webhook/deployment.yaml

```diff
@@ -10,7 +10,10 @@ spec:
   selector:
     matchLabels:
       name: shared-resource-csi-driver-webhook
-  strategy: {}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
```

Comment:
I did not set maxSurge, because running more replicas of the webhook for a short while is not really a bad thing - the webhook does not use node ports.